aiida/orm/nodes/data/code/abstract.py

```diff
@@ -70,12 +71,38 @@ def can_run_on_computer(self, computer: Computer) -> bool:
         """
 
     @abc.abstractmethod
-    def get_executable(self) -> str:
-        """Return the executable that the submission script should execute to run the code.
+    def get_executable(self) -> pathlib.Path:
+        """
+        Return the executable that the submission script should execute to run the code.
```

Comment:
PEP 257 docstring guidelines say to keep the first line on the same line as the opening quotes. Just a minor thing but good to keep the consistency when adding new code.
```suggestion
        """Return the executable that the submission script should execute to run the code.
```

Reply:
I see, didn't know that. I thought you are too lazy to add a new line 😆